### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mwenyoa**
### In this project milestone we did the following:

- Created a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. `decode_char(".-")` returns `"A"`).
- Created a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. `decode_word("-- -.--")` returns `"MY"`).
- Created a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.
    ```
      decode("-- -.--   -. .- -- .")
    ```
  
  returns `"MY NAME"`).
- Reused methods whenever possible. Make sure that your code is DRY.
- Once our code was ready we used it to decode the message from the old bottle:

```
      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...
```

### CODE REVIEW

File: decorder.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          sentence_converted=''
+          sentence.split('   ').each do |text|
+             dc= convert_word(text)
+             sentence_converted +="#{dc} "
+          end
+      end
+      sentence_converted
+  end
+  # Testing
+    puts convert_sentence('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```
Comment (julie-ify) [inline decorder.rb]: - [ ] Good job so far, however, there is a trailing whitespace at the end of the sentences which is not supposed to be there. Kindly fix the trailing whitespace by just adding `.strip` to the `sentence_converted` on line `67` to be `sentence_converted.strip` .

Your Implementation | When added `.strip`
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @mwenyoa and @GedeonTS,

Good job so far!

## Highlights!
- GitFlow was followed ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: decorder.rb (hunk 1 above)
Comment (GedeonTS) [reply decorder.rb]: Got it, thanks! :smile:

**Review by BenMukebo — APPROVED** [review]
### Hi @mwenyoa, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**